src/google/protobuf/compiler/cpp/cpp_message.cc

```diff
@@ -253,6 +253,10 @@ void CollectMapInfo(const Options& options, const Descriptor* descriptor,
   std::map<std::string, std::string>& vars = *variables;
   const FieldDescriptor* key = descriptor->FindFieldByName("key");
   const FieldDescriptor* val = descriptor->FindFieldByName("value");
+  // Can`t find the 'key'
```

Comment:
Could you change these to use `GOOGLE_CHECK_NOTNULL()`? I think you can also remove the comments since the checks are fairly self-explanatory.